requirements/default.txt

```diff
@@ -6,4 +6,4 @@ click>=8.0.3,<9.0
 click-didyoumean>=0.0.3
 click-repl>=0.2.0
 click-plugins>=1.1.1
-setuptools>=59.1.1,<59.7.0
```

Comment:
why setuptools is removed?

Reply:
Because it's no longer required to run the lib. Usage of pkg_resources (that is provided by setuptools) was replaced with usage of importlib.metadata.